### ISSUE TITLE
Rule ID checkboxgroup

### ISSUE THREAD

**Issue by devarshipant** [dequelabs/axe-core#557]
When using same name attributes on checkbox controls, and aria-labelledby associating the pattern label with **only** the first checkbox control, rule checkboxgroup throws a critical violation. Is that required? From an a11y standpoint, is there anything wrong in associating the pattern label with only the first checkbox.
<img width="692" alt="checkboxgroup" src="https://user-images.githubusercontent.com/7377040/31144847-e3c2f302-a84f-11e7-9721-358c4f08cec4.PNG">

See attached screenshot.

**Comment by WilcoFiers** [comment]
I don't quite follow. Can you put a code example on codepen or something?

**Comment by devarshipant** [comment]
Try this out. thanks!
[L1 checkbox - same name attr first lbdby-IdealState.txt](https://github.com/dequelabs/axe-core/files/1356201/L1.checkbox.-.same.name.attr.first.lbdby-IdealState.txt)
